modules/theme-tailwind/organisms/CrossSellModal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <Modal @close="emit('close')">
+  <Modal v-if="data?.products.length" @close="emit('close')">
```

Comment:
This change will cause the modal to not appear if there are no cross-sell products.